os_migrate/plugins/module_utils/const.py

```diff
@@ -15,4 +15,5 @@
 RES_TYPE_SECURITYGROUP = 'openstack.network.SecurityGroup'
 RES_TYPE_SERVER = 'openstack.compute.Server'
 RES_TYPE_SUBNET = 'openstack.subnet.Subnet'
+RES_TYPE_USER = 'openstack.subnet.User'
```

Comment:
is ‘subnet’ a typo here since you’re using the identity.v3.user in user.py?

Reply:
Yes sorry a typo, thanks for the check